keyboards/kbdfans/maja_soldered/keymaps/via/keymap.c

```diff
@@ -0,0 +1,17 @@
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
All 4 layers need to be populated for the via keymap, unless the number of layers has been decreased.